src/pages/TrueSight/hooks/useGetTrendingSoonTokenId.ts

```diff
@@ -29,7 +29,7 @@ export default function useGetTrendingSoonTokenId(token?: Token): number | undef
     }
 
     asyncF()
-  }, [token])
+  }, [JSON.stringify(token)])
```

Comment:
token0, token1 were memorize by useMemo, so why do you need stringify here?

Reply:
I know, but somehow it just re-render... It takes time to find the root causes.

Reply:
I solve it.